src/index.js

```diff
@@ -183,6 +194,12 @@ class HCaptcha extends React.Component {
       hcaptcha.reset(captchaId) // If hCaptcha runs into error, reset captcha - hCaptcha
 
       if (onExpire) onExpire();
+
+      // Reject promise if exists
+      if (typeof this.reject !== undefined) {
+        this.reject(new Error("hCaptcha expired"));
```

Comment:
Should this be handled like so or be handled at all?

Reply:
This looks correct: reject with new error and message.